.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
Should we consider ignoring IDE folders?

https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore